hardware/Yeelight.cpp

```diff
@@ -283,12 +283,12 @@ bool Yeelight::WriteToHardware(const char *pdata, const unsigned char length)
 		message = ss.str();
 		break;
 	case Limitless_SetRGBColour: {
-			sendOnFirst = true;
-			float cHue = (359.0f / 255.0f)*float(pLed->value); // hue given was in range of 0-255
-			ss << "{\"id\":1,\"method\":\"set_hsv\",\"params\":[" << cHue << ", 100, \"smooth\", 2000]}\r\n";
-			message = ss.str();
-		}
-		break;
+		sendOnFirst = true;
+		float cHue = (359.0f / 255.0f)*float(pLed->value); // hue given was in range of 0-255
+		ss << "{\"id\":1,\"method\":\"set_hsv\",\"params\":[" << cHue << ", 100, \"smooth\", 2000]}\r\n";
+		message = ss.str();
+	}
+								 break;
```

Comment:
Please do not submit changes for this file, as it is not related to this pull request